src/line_launcher/mod.rs

```diff
@@ -112,15 +120,27 @@ impl LineLauncher {
             NoteOffTriggerer::new(midi_message_sender.clone(), state_mutex.clone());
         note_off_triggerer.listen();
         let mut duration_between_sixteenth_notes = DurationBetweenSixteenthNotes::new();
-        let mut duration_ratio = 1.0;
         let mut midi_message_bus = Bus::new(100);
-        let duration_ratio_receiver =
-            listen_for_duration_control_changes(midi_message_bus.add_rx());
-        thread::spawn(move || {
-            for midi_message in midi_messages.iter() {
-                midi_message_bus.broadcast(midi_message);
-            }
-        });
+        let (mut duration_ratio, duration_ratio_receiver) = match duration_ratio_slider {
```

Comment:
The objective here was to not do ratio-based note-off'ing unless the slider for it is specified in the config